NDB.Covid19/NDB.Covid19/Locales/pl.json

```diff
@@ -1,12 +1,12 @@
 {
-  "LAUNCHER_PAGE_OS_VERSION_DIALOG_MESSAGE_IOS": "Aplikacja Smittestopp działa tylko w systemie iOS 13.6 lub nowszym. Aktualizuj do nowszej wersji. Zrobisz to na swoim telefonie, wchodząc na Ustawienia -> Ogólne -> Aktualizacje aplikacji systemowych.",
+  "LAUNCHER_PAGE_OS_VERSION_DIALOG_MESSAGE_IOS": "Aby korzystać ze Smittestopp, musisz zaktualizować swój telefon. Wejdź w Ustawienia -> Ogólne -> Zaktualizuj Oprogramowanie.",
```

Comment:
This is not a correct text in Polish iPhone, I sent the correct one in the email

Reply:
Aby korzystać z aplikacji Smittestopp musisz zaktualizować swój telefon. Ustawienia -> Ogólne -> Uaktualnienia

Reply:
@Michael-Olesen please add above to both excel sheet and this .json